src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.io.IOException;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class MainApp {
+    public static void main(String[] args) throws IOException {
+        WriteToFile testRecord = new WriteToFile();
+        testRecord.csvFileWriter("current_reminder.csv", "test11.csv");
+        testRecord.csvFileWriter("current_reminder.csv", "test11.csv");
+        System.out.println(Storage.fruitsInStore.toString());
```

Comment:
To my opinion when I run this app I should be proposed to specify path to data file. Path shouldn't be hardcoded.

Reply:
done